src/AppFooter.js

```diff
@@ -0,0 +1,14 @@
+function AppFooter({ changeButton, buttonClass, addToStorage}) {
+    return (
+        <div className='footer'>
+            <button
+                className={buttonClass}
+                id='saveButton'
```

Comment:
it's a bad idea to use hardcoded html id's inside of React component, please don't use it

Reply:
Ok, I removed it.